spec/07-implicits.md

```diff
@@ -61,7 +61,7 @@ call without a prefix and that denote an
 [implicit definition](#the-implicit-modifier)
 or an implicit parameter.  An eligible
 identifier may thus be a local name, or a member of an enclosing
-template, or it may be have been made accessible without a prefix
+template, or it may be made accessible without a prefix
```

Comment:
The original tense is correct but remove the extraneous "be".

Or edit the list: "To be accessible without a prefix, an identifier must be a local name, a member of an enclosing template or a name introduced by a import clause." But "member of an enclosing template" is imprecise because of package objects.